clang/test/dpct/replace_callee_name_only.yaml

```diff
@@ -0,0 +1,24 @@
+- Rule: rule0
+  Kind: API
+  Priority: Takeover
+  In: cub::DeviceReduce::Sum
+  Out: REPLACE_CALLEE_NAME_ONLY
```

Comment:
let's use a meaningful API here,  in case user want to build the lit case.

Reply:
done